src/style.css

```
@@ -0,0 +1,72 @@
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.logo-box {
+  width: 40px;
+  height: 40px;
+}
+
+.logo-box img {
+  width: 100%;
+  height: 100%;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+.nav-item {
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-of-type(1) {
+  margin-right: 0;
+}
```

Instead two selectors you can create only one:
```suggestion
.nav-item:not(:first-child) {
  // code
}
```

---

.nav-item:not(:first-child)  - select all without first. It is error

---

Thanks. I did not immediately understand